Modules/_testcapi/parts.h

```diff
@@ -1,27 +1,9 @@
 #ifndef Py_TESTCAPI_PARTS_H
 #define Py_TESTCAPI_PARTS_H
 
-#include "pyconfig.h"  // for Py_TRACE_REFS
-
-// Figure out if Limited API is available for this build. If it isn't we won't
-// build tests for it.
-// Currently, only Py_TRACE_REFS disables Limited API.
-#ifdef Py_TRACE_REFS
-#undef LIMITED_API_AVAILABLE
-#else
-#define LIMITED_API_AVAILABLE 1
```

Comment:
This flag is still referenced in `Modules/_testcapi/heaptype_relative.c`, `Modules/_testcapi/vectorcall_limited.c`, `Modules/_testcapimodule.c`, which means the limited API tests are now being incorrectly skipped unconditionally.

- #109045